### PR TITLE
[UIPO-4] Application global styles setup

### PR DESCRIPTION
**Pull request by castan1c**

### CODE REVIEW

File: src/app/containers/Counter/CounterContainer.js [hunk 1]
```diff
@@ -1,14 +1,7 @@
...
-=======
-import { connect }  from 'react-redux';
 import { increment, decrement } from '../../actions';
 import Counter from '../../components/Counter/Counter';
-import { bindActionCreators } from 'redux'; 
->>>>>>> ad871e26a70080ead7260fd3c52f46efbfb84e9e:src/app/containers/Counter/CounterContainer.js
+import { bindActionCreators } from 'redux';
 
 const mapStateToProps = (state) => {
   return {
```
Comment (Hollister009) [inline src/app/containers/Counter/CounterContainer.js:7]: ???

Comment (castan1c) [reply src/app/containers/Counter/CounterContainer.js:7]: had to resolve conflicts...

Comment (Hollister009) [reply src/app/containers/Counter/CounterContainer.js:7]: Oh I see, but why this was even merged?

File: src/app/containers/ProductDetailsPage/ProductDetailsPage.js [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
+//       <div>
+//         <div>{this.state.product.name}</div>
+//         <div>{this.state.product.id}</div>
+//         <button onClick={() => this.returnBack()}>Go Back</button>
+//       </div>
+//     );
+//   }
+// }
+
+// export default ProductDetailsPage;
```
Comment (Hollister009) [inline src/app/containers/ProductDetailsPage/ProductDetailsPage.js:31]: I don't get why this file was commented out?

File: src/app/index.html [hunk 3]
```diff
@@ -7,6 +7,6 @@
   </head>
   <body>
     <div id="root"></div>
-    <link rel="stylesheet" href="/Users/mtorskyi/Documents/project/ui-internship/dist/main.min.css">
+    <link rel="stylesheet" href="./css/main.min.css">
```
Comment (Hollister009) [inline src/app/index.html:10]: @TorskyM why did you hardcoded link to styles?

File: src/app/service/httpService.js [hunk 4]
```diff
@@ -1,60 +1,55 @@
...
-    catch (error) {
+    } catch (error) {
       return error.response.data;
     }
   }
 };
 
- 
\ No newline at end of file
+
```
Comment (Hollister009) [inline src/app/service/httpService.js:55]: I don't like this approach to commit unrelated files in one PR

**Review by Hollister009 — CHANGES_REQUESTED** [review]
You can do better! I can consider this PR as draft

File: src/app/containers/Counter/CounterContainer.js (hunk 1 above)
Comment (castan1c) [reply src/app/containers/Counter/CounterContainer.js:7]: @Nazar-bnm merged 3 PRs today, we got a lot of conflicts after that.

File: src/app/service/httpService.js (hunk 4 above)
Comment (castan1c) [reply src/app/service/httpService.js]: my bad, I will redo the PR only with files that are related to my task

File: src/app/containers/ProductDetailsPage/ProductDetailsPage.js (hunk 2 above)
Comment (castan1c) [reply src/app/containers/ProductDetailsPage/ProductDetailsPage.js:31]: our config requires propTypes now, i can't push without it

File: src/app/service/httpService.js (hunk 4 above)
Comment (castan1c) [reply src/app/service/httpService.js:55]: @Hollister009 The thing is if you pull development right now - you can't push it without fixing everything. 3 PR's where merged when we had no eslint/hooks/ etc.

Comment (castan1c) [reply src/app/service/httpService.js:55]: @Hollister009 I created a new PR https://github.com/Nazar-bnm/ui-internship/pull/6: created a new branch from development, added files from my task only and pushed it with ```--no-verif```. If it's suitable for you i will close this PR

**Comment by Hollister009** [comment]
Too many changes, closing
